lib/puppetlabs_spec_helper/rake_tasks.rb

```diff
@@ -324,6 +324,22 @@ def max_thread_limit
   Rake::Task[:spec_clean].invoke
 end
 
+begin
+  require 'parallel_tests'
+
+  args = ['-t', 'rspec']
+  Dir.glob(pattern).each { |f| args << f }
+
+  desc "Parallel spec tests"
+  task :parallel_spec do
+    Rake::Task[:spec_prep].invoke
+    ParallelTests::CLI.new.run(args)
+    Rake::Task[:spec_clean].invoke
+  end
+rescue LoadError
+  # ignore
```

Comment:
In the case that the gem is not installed, I'd prefer an empty task that reminds the user that the gem is not installed and causes a failure. That way misconfigured CI environments fail consistently and helpfully.